daemon/daemon.go

```diff
@@ -779,7 +779,7 @@ func NewDaemon(dp datapath.Datapath) (*Daemon, *endpointRestoreState, error) {
 	debug.RegisterStatusObject("ipam", d.ipam)
```

Comment:
Commit misses sign-off

Reply:
Is this a comment for me?

Reply:
This is resolved now that all commits are rebased into one commit with a sign-off.